Jenkinsfile

```diff
@@ -348,7 +348,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
     "PATH=${path}",
     "PYTHON_ENV=${pythonEnv}",
     "RACE_DETECTOR=true",
-    "TEST_COVERAGE=true",
+    //"TEST_COVERAGE=true", TODO
```

Comment:
What's the reason?

Reply:
It was just a test to see if the issue still persist if the coverpkg flag is removed.